src/types.md

```diff
@@ -498,18 +498,25 @@ Because captures are often by reference, the following general rules arise:
 
 ## Trait objects
 
+> **<sup>Syntax</sup>**  
+> _TraitObjectType_ :  
+> &nbsp;&nbsp; [_LifetimeOrPath_] ( `+` [_LifetimeOrPath_] )<sup>\*</sup> `+`<sup>?</sup>
+>
+> _LifetimeOrPath_ :
+> &nbsp;&nbsp; [_Path_] | [_LIFETIME_OR_LABEL_]
+
 A *trait object* is an opaque value of another type that implements a set of
 traits. The set of traits is made up of an [object safe] *base trait* plus any
 number of [auto traits].
 
 Trait objects implement the base trait, its auto traits, and any super traits
 of the base trait.
 
-Trait objects are written as the path to the base trait followed by the list
-of auto traits followed optionally by a lifetime bound all separated by `+`. For
-example, given a trait `Trait`, the following are all trait objects: `Trait`,
-`Trait + Send`, `Trait + Send + Sync`, `Trait + 'static`,
-`Trait + Send + 'static`.
+Trait objects are written the same as trait bounds with the exception that all
```

Comment:
I think this would be clearer as separate sentences instead of "with the exception that".